### PR TITLE
Improved code readability for P/Invoke

### PR DESCRIPTION
**Pull request by farag2**
Also it will be awseome if I you with a CI/CD config to automate the process of attaching archives to the release page. Also `lzma` cannot be opened via the File Explorer. The standard is `zip`.
Share your thoughts please.

### CODE REVIEW

**Comment by pbatard** [comment]
I'll look at it, but please understand that in the context for which I actually develop Fido (which is pretty much exclusively for usage with Rufus), it might be a while, especially as I am going to have to validate that this doesn't break stuff on Windows 8 platforms.

The context is also the reason it's a `.lzma`, because, 1. it compresses better than zip and 2. this makes it easier to download from Rufus, which already has support for LZMA decompression. Also, 7-zip has no issue opening such files and, with the default compression support from Windows being years behind what it should be, I would really encourage people to use a much better handler, such as 7-zip, for compressed files.

In short, my actual goal with this project is definitely not the goal that you expect it to have, when it comes to being standalone or supporting bare Windows installations...

Still, I'll see what I can do with this PR.

**Comment by farag2** [comment]
> this doesn't break stuff on Windows 8 platforms

It won't. I just improved the readability, not the code. :) <https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_splatting>

**Comment by pbatard** [comment]
Sorry but I'll take _testing_ over _"trust me"_ when it comes to validating that a script that I provide to users does not introduce any regression...

**Comment by farag2** [comment]
OK, no problem. But there's a tab with changes highlighted with green. :)
